src/test/kotlin/io/bazel/kotlin/KotlinAssertionTestCase.kt

```diff
@@ -120,7 +120,7 @@ abstract class BasicAssertionTestCase {
         ProcessBuilder().command("bash", "-c", Paths.get(executable).fileName.toString())
             .also { it.directory(executable.resolveDirectory()) }
             .start().let {
-                it.waitFor(5, TimeUnit.SECONDS)
+                it.waitFor(10, TimeUnit.SECONDS)
```

Comment:
This just accounts for the e2e test now running a test with a 2-second delay in the production code.